src/main.rs

```diff
@@ -108,7 +108,40 @@ fn main() {
     let _ = ansi_term::enable_ansi_support();
     logger::init();
 
-    let args = Cli::parse();
+    let args = match Cli::try_parse() {
+        Ok(args) => args,
+        Err(e) => {
+            let is_info_only = matches!(
+                e.kind,
+                clap::ErrorKind::DisplayHelpOnMissingArgumentOrSubcommand
+                    | clap::ErrorKind::DisplayHelp
+                    | clap::ErrorKind::DisplayVersion
+            );
```

Comment:
clap uses this check internally:
```suggestion
            let is_info_only = !e.use_stderr();
```

Should be slightly more reliable. Beyond this LGTM.